### PR TITLE
New tests for min-width:auto and items with aspect ratio

### PR DESCRIPTION
**Pull request by svillar**
A couple of new tests verifying that we do not incorrectly stretch flex items with aspect ratio due to min-width:auto resolution.

### CODE REVIEW

**Comment by svillar** [comment]
JFTR these tests come from https://bugs.webkit.org/show_bug.cgi?id=221110. WebKit was incorrectly enlarging the size of some flex items with aspect ratio due to some incorrect implementation of `min-size:auto`.

File: css/css-flexbox/flexbox-min-width-auto-006.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    background-color: red;
+    width: 10px;
+    height: 10px;
+    top: 40px;
+    z-index: -1;
+}
+.outer {
+    width: 100px;
+    display: flex;
+    /*background: #eee;*/
```
Comment (mrego) [inline css/css-flexbox/flexbox-min-width-auto-006.html:20]: Nit: Remove this line.

File: css/css-flexbox/flexbox-min-width-auto-006.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    border: 1px solid black;
+}
+.inner {
+    display: flex;
+    height: 100px;
+}
+</style>
+
+<p>Test passes if there are a 20x20 and a 60x100 green boxes enclosed on each 100x100 square.</p>
```
Comment (mrego) [inline css/css-flexbox/flexbox-min-width-auto-006.html:29]: I'd also mention they'll be vertically centered (basically the 20x20 one).

File: css/css-flexbox/flexbox-min-width-auto-005.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+<style>
+#reference-overlapped-red {
+    position: absolute;
+    background-color: red;
+    width: 100px;
+    height: 50px;
+    z-index: -1;
+}
+#constrained-flex {
```
Comment (mrego) [inline css/css-flexbox/flexbox-min-width-auto-005.html:16]: Nit: Better use a class as the "id" is repeated on the test.

File: css/css-flexbox/flexbox-min-width-auto-006.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+<style>
+#reference-overlapped-red {
+    position: relative;
+    background-color: red;
+    width: 10px;
+    height: 10px;
+    top: 40px;
+    z-index: -1;
+}
+.outer {
```
Comment (mrego) [inline css/css-flexbox/flexbox-min-width-auto-006.html:17]: Better use names like in the previous test "constrained-width-flex" or something like that.

File: css/css-flexbox/flexbox-min-width-auto-005.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<title>CSS Flexible Box Test: Aspect ratio handling of images</title>
+<link rel="author" title="Sergio Villar Senin" href="mailto:svillar@igalia.com" />
+<link rel="help" href="https://drafts.csswg.org/css-flexbox-1/#min-size-auto" />
+<link rel="match" href="reference/flexbox-min-width-auto-005-ref.html" />
+<meta name="assert" content="Test that min-width:auto does not incorrectly stretch items with aspect ratio" />
+
+<style>
+#reference-overlapped-red {
```
Comment (mrego) [inline css/css-flexbox/flexbox-min-width-auto-005.html:9]: Nit: Better use a class as the "id" is duplicated in the test.

**Review by mrego — APPROVED** [review]
LGTM, but I added a few inline comments and suggestions.

**Comment by svillar** [comment]
A couple of new tests verifying that we do not incorrectly stretch flex items with aspect ratio due to min-width:auto resolution.

**Pull request merged**